examples/advanced/xgboost/histogram-based/jobs/base/app/config/config_fed_client.json

```diff
@@ -1,23 +1,24 @@
 {
   "format_version": 2,
+  "num_rounds": 100,
   "executors": [
     {
       "tasks": [
         "train"
       ],
       "executor": {
         "id": "Executor",
-        "name": "FedXGBHistogramExecutor",
+        "path": "nvflare.app_opt.xgboost.histogram_based.executor.FedXGBHistogramExecutor",
         "args": {
           "data_loader_id": "dataloader",
-          "num_rounds": 100,
+          "num_rounds": "{num_rounds}",
           "early_stopping_rounds": 2,
           "xgb_params": {
             "max_depth": 8,
             "eta": 0.1,
             "objective": "binary:logistic",
             "eval_metric": "auc",
-            "tree_method": "gpu_hist",
+            "tree_method": "hist",
```

Comment:
users can modify https://github.com/NVIDIA/NVFlare/blob/main/examples/advanced/xgboost/prepare_job_config.sh to change the algo.

Reply:
actually, just found out the "gpu_hist" is deprecated since xgboost 2.0.0: https://github.com/dmlc/xgboost/releases/tag/v2.0.0, I will change all things to hist in this PR or the metric callback PR.